judo-tatami-jsl-jsl2psm/src/main/epsilon/transformations/psm/modules/structure/transferDeclarationTransferAttribute.etl

```diff
@@ -57,8 +63,62 @@ rule CreateMappedTransferAttribute
+// Primitive and Identifier default value
+////////////////////////////////////////////////////////////////////////////////////////////
+@greedy
+rule CreateMappedTransferAttributeEntityDefault
+    transform s : JSL!TransferFieldDeclaration
+    to t : JUDOPSM!TransferAttribute
+    extends AbstractCreateTransferAttributeForTransferFieldDeclaration {
+      guard: s.referenceType.isKindOf(JSL!PrimitiveDeclaration) and s.maps.isDefined() 
+        and s.maps.features.first.member.defaultExpression.isDefined() and not JSL!TransferDefault.all().exists(d | d.field.reference == s)
+      t.name = defaultDefaultNamePrefix + s.name + defaultDefaultNameMidfix + s.eContainer.name + defaultDefaultNamePostfix;
+      t.setId("(jsl/" + s.getId() + ")/CreateMappedTransferAttributeEntityDefault");      
+      t.binding = s.maps.features.first.member.defaultExpression.equivalent("CreateDefaultValueForPrimitiveEntityMember");               
+      s.eContainer.getPSMEquivalent().attributes.add(t);
+      log.debug("Created TransferAttribute DefaultValue (Field) for EntityDefault: [" + t.name + "] into [" + t.eContainer.name + "]");
+}
+
+
+@greedy
+rule CreateTransferEntityDefaultValueAttributeForMappedTransferObjectConstructor
+    transform s: JSL!TransferDefault
+    to t: JUDOPSM!TransferAttribute {
+        guard: s.field.reference.referenceType.isKindOf(JSL!PrimitiveDeclaration) and s.field.reference.eContainer.map.isDefined()
+        t.name = defaultDefaultNamePrefix + s.field.reference.name + defaultDefaultNameMidfix + s.field.reference.eContainer.name + defaultDefaultNamePostfix;
+        t.binding = s.equivalent("CreateDefaultDataPropertyForMappedTransferObjectConstructor");	  
+        t.dataType = s.field.reference.referenceType.getPSMEquivalent();
+        t.setId("(jsl/" + s.field.reference.getId() + ")/CreateTransferEntityDefaultValueAttributeForMappedTransferObjectConstructor");
+        s.eContainer.eContainer.getPSMEquivalent().attributes.add(t);
+        log.debug("Created TransferAttribute DefaultValue (Field) for MappedTransferObjectConstructor: [" + t.name + "] into [" + t.eContainer.name + "]");
+}
+
+@greedy
+rule CreateTransferEntityDefaultValueAttributeForUnmappedTransferObjectConstructor
+    transform s: JSL!TransferDefault
+    to t: JUDOPSM!TransferAttribute {
+        guard: s.field.reference.referenceType.isKindOf(JSL!PrimitiveDeclaration) and s.field.reference.eContainer.map.isUndefined()
+        t.name = defaultDefaultNamePrefix + s.field.reference.name + defaultDefaultNameMidfix + s.field.reference.eContainer.name + defaultDefaultNamePostfix;
+        t.binding = s.equivalent("CreateDefaultStaticDataForUnmappedTransferObjectConstructor");	  
+        t.dataType = s.field.reference.referenceType.getPSMEquivalent();
+        t.setId("(jsl/" + s.field.reference.getId() + ")/CreateTransferEntityDefaultValueRelationForUnmappedTransferObjectConstructor");
```

Comment:
> CreateTransferEntityDefaultValue**Relation**ForUnmappedTransferObjectConstructor

Should be CreateTransferEntityDefaultValue**Attribute**ForUnmappedTransferObjectConstructor